helmfile/values/kube-prometheus-stack-sc.yaml.gotmpl

```
@@ -127,9 +127,12 @@ alertmanager:
     ## Time duration Alertmanager shall retain data for. Default is '120h', and must match the regular expression
     ## [0-9]+(ms|s|m|h) (milliseconds seconds minutes hours).
     ##
+    replicas: {{ .Values.prometheus.alertmanagerSpec.replicas }}
     retention: {{ .Values.prometheus.retention.alertmanager }}
     resources: {{- toYaml .Values.prometheus.alertmanagerSpec.resources | nindent 6 }}
     storage: {{- toYaml .Values.prometheus.alertmanagerSpec.storage | nindent 6 }}
+    topologySpreadConstraints:
+    {{- toYaml .Values.prometheus.alertmanagerSpec.topologySpreadConstraints | nindent 4  }}
```

Should follow the same format as the rest i would say
```suggestion
    topologySpreadConstraints: {{- toYaml .Values.prometheus.alertmanagerSpec.topologySpreadConstraints | nindent 4  }}
```